compiler.rb

```diff
@@ -142,6 +142,8 @@
     provider = \
       override_providers[force_provider].new(provider_config, product_api)
   end
+
   provider.generate output_path, types_to_generate, version
+  provider.copy_common_files output_path, version
```

Comment:
This is not really your issue, but I would really like to add parentheses to all of these. Adding mention so I remember to do this (@emilymye). 

something something just because we can